grafana-plugin/e2e-tests/outgoingWebhooks/simpleWebhook.test.ts

```diff
@@ -0,0 +1,78 @@
+import express from 'express';
+
+import { expect, test } from '../fixtures';
+import { EscalationStep, createEscalationChain } from '../utils/escalationChain';
+import { clickButton, generateRandomValue } from '../utils/forms';
+import { createIntegrationAndSendDemoAlert } from '../utils/integrations';
+import { goToOnCallPage } from '../utils/navigation';
+import { checkWebhookPresenceInTable } from '../utils/outgoingWebhooks';
+
+const createWebhook = async ({ page, webhookName, webhookUrl }) => {
+  await goToOnCallPage(page, 'outgoing_webhooks');
+
+  await clickButton({ page, buttonText: 'New Outgoing Webhook' });
+
+  await page.getByText('Simple').first().click();
+
+  await page.waitForTimeout(2000);
+
+  await page.keyboard.insertText(webhookUrl);
+  await page.locator('[name=name]').fill(webhookName);
+  await page.getByLabel('New Outgoing Webhook').getByRole('img').nth(1).click(); // Open team dropdown
+  await page.getByLabel('Select options menu').getByText('No team').click();
+  await clickButton({ page, buttonText: 'Create' });
+};
+
+test.describe('simple webhook', () => {
+  test('Create and check it is displayed on the list correctly', async ({ adminRolePage: { page } }) => {
+    const webhookName = generateRandomValue();
+    await createWebhook({ page, webhookName, webhookUrl: 'https://example.com' });
+    await checkWebhookPresenceInTable({ page, webhookName, expectedTriggerType: 'Escalation step' });
+  });
+
+  test('Create and check that our webhook actually receives the payload', async ({ adminRolePage: { page } }) => {
+    const escalationChainName = generateRandomValue();
+    const integrationName = generateRandomValue();
+    const PORT = 5050;
+
+    /**
+     * This is a simple express server that listens for outgoing webhook requests
+     * The backend is able to communicate with it via host.docker.internal (docker -> host communication)
+     */
+    let resolveRequest: (value: unknown) => void;
+    const requestPromise = new Promise<any>((resolve) => {
+      resolveRequest = resolve;
+    });
+
+    const app = express();
+    app.use(express.json());
+    app.post('/', (req, res) => {
+      resolveRequest(req.body); // Resolve the promise with the request body
+      res.send('ok');
+    });
+    app.listen(PORT);
+
+    /**
+     * TODO: this might need to be parametrized to be read from an env var
+     * rather than hardcoding the hostname to be host.docker.internal
+     */
+    const webhookUrl = `http://host.docker.internal:${PORT}`;
```

Comment:
tried using `ngrok` here.. this worked locally, but it seems like there was some issues on CI, with the API being able to communicate with the `express` server's public internet URL assigned by `ngrok` (see test failure [here](https://github.com/grafana/oncall/actions/runs/8271238688/job/22630552253#step:19:26)).

See [this](https://raintank-corp.slack.com/archives/CEXAGGYVC/p1710432296988059) Slack conversation for more context